lib/tasks/valid_test_data_generator/npq_lead_provider_populater.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require "tasks/school_urn_generator"
+require "tasks/trn_generator"
+require "active_support/testing/time_helpers"
+require "tasks/valid_test_data_generator/base_populater"
+
+module ValidTestDataGenerator
+  class NPQLeadProviderPopulater < BasePopulater
+    def populate
+      return unless Rails.env.in?(%w[development review sandbox])
+
+      logger.info "NPQLeadProviderPopulater: Started!"
+
+      ActiveRecord::Base.transaction do
+        generate_new_schools!
```

Comment:
maybe we rename this to show that it is also generating participants as well as schools?